bin/quack

```diff
@@ -0,0 +1,3 @@
SCO110' =>'===DELETED===','SCO120' =>'Duplicated declaration of `%s\' on where clause','SCO130' =>'Symbol `%s\' declared twice','SCO140' =>'Called %s outside a loop','SCO150' =>'Called %s with undefined label `%s\'','SCO160' =>'Called %s with invalid label `%s\'','SCO170' =>'Duplicated declaration of `%s\' for enum `%s\'','SCO180' =>'Same variable name `%s\' for key and value on foreach','SCO190' =>'%s `%s\' not found in implementation','SCO200' =>'`%s\' is not a class in implementation','SCO210' =>'===DELETED===','SCO220' =>'===DELETED===','TYP010' =>'Value passed to while loop should be a `boolean\', not a `%s\'','TYP020' =>'Cannot add element of type `%s\' to array of `%s\'','TYP030' =>'The type of post conditional statement should be a `boolean\', not a `%s\'','TYP040' =>'Expected index of array to be a number. Got `%s\'','TYP050' =>'Expected index of map to be a `%s\'. Got `%s\'','TYP060' =>'Trying to access by index an element of type `%s\' that is not accessible','TYP070' =>'Key on index %d of map expected to be `%s\'. Got `%s\'','TYP080' =>'Value on index %s of map expected to be `%s\'. Got `%s\'','TYP090' =>'Expression of type `%s\' has no property `%s\'','TYP100' =>'Trying to set value of type `%s` to %s','TYP110' =>'No type overload found for operator `%s\' at {%s %s %s}','TYP120' =>'===DELETED===','TYP130' =>'Why in the world are you trying to compare two expressions of different types? at {%s %s %s}','TYP140' =>'Value passed to if statement should a `boolean\', not a `%s\'','TYP150' =>'Expecting type of case of switch statement to be `%s\'. Got `%s\'','TYP160' =>'More than one else clause for switch statement','TYP170' =>'Expecting type of field `%s\' of foreach statement to be a `number\'. Got `%s\'','TYP180' =>'Condition passed to elif statement should be a `boolean\', not a `%s\'','TYP190' =>'Cannot infer type of non-variable `%s\'','TYP200' =>'Expected condition %d of when expression to be a `boolean\'. Got `%s\'','TYP210' =>'All results to when expression must be of same type `%s\', but result %d is `%s\'','TYP220' =>'Expected a `number\' on operand `%s\' of range expression. Got `%s\'','TYP230' =>'No type overload for operator `%s\' on type `%s\'','TYP240' =>'Condition of ternary operator should de `boolean\'. Got `%s\'','TYP250' =>'Both sides of ternary expression must have the same type. Got `%s\' and `%s\'','TYP260' =>'`%s\' is not iterable','TYP270' =>'You forgot to set a value for immutable variable `%s\', my love','TYP280' =>'Found cyclic reference for type `%s\'','TYP290' =>'Caught declaration of free variable `%s\'','TYP300' =>'Cannot assign to variable `%s\' of type `%s\' a value of type `%s\'','TYP310' =>'I\'m sorry, but type `%s\' is not callable','TYP320' =>'Function of type `%s\' expected %d arguments, but received %d','TYP330' =>'Argument number %d of call expected to be a `%s\', but it is a `%s\'','TYP340' =>'On map, cannot match the type of %s `%s\' with `%s\'','TYP350' =>'Bad equation: `%s\' and `%s\', because:
```

Comment:
Are you sure about this file here? I don't see its usefulness once it is Quack's compact version (and Quack will change a lot, making it infeasible)

Reply:
Yes, although we don't need to update it every time, this compiled file makes it possible installing Quack in milliseconds!